### PR TITLE
Enhance BulkImport pre_cook

### PR DESCRIPTION
**Pull request by quangbuule**
Input:
```
le_quang_buu
foo
bar
```
Output of previous behavior:

> le<em>quang</em>buu foo bar

Output of new behavior (preventing emphasizing inside words and converting `\n` into `<br>\n`):

> le_quang_buu
> foo
> bar

### CODE REVIEW

**Comment by quangbuule** [comment]
https://github.com/quangbuule/discourse/blob/bulk-import-patch-2/script/bulk_import/base.rb#L580

That line should be:
```ruby
```

File: script/bulk_import/base.rb [hunk 1]
```diff
@@ -579,8 +580,11 @@ def pre_cook(raw)
     cooked = @markdown.render(cooked).scrub.strip
 
     cooked.gsub!(/\[QUOTE="?([^,"]+)(?:, post:(\d+), topic:(\d+))?"?\](.+?)\[\/QUOTE\]/im) do
-      username, post_id, topic_id = $1, $2, $3
-      quote = @markdown.render($4.presence || "").scrub.strip
+      username, post_id, topic_id, quote = $1, $2, $3, $4
+
+      quote = quote.scrub.strip
```
Comment (ZogStriP) [inline script/bulk_import/base.rb]: Why did you remove the markdown rendering? Sometimes, a quote has markdown/html and we need to render it.

Comment (quangbuule) [inline script/bulk_import/base.rb]: In my opinion:
When it's html, we don't need to do markdown.
When it's markdown, we already rendered just few lines upper (in L580).

Could you send me some cases we need to render again?

**Comment by SamSaffron** [comment]
@ZogStriP is on holidays at the moment, as soon as he is back in a week or so he will decide what to do here. Sorry about the delay.

**Comment by ZogStriP** [comment]
Ok, let's try it.
